### ISSUE TITLE
Paste events function not working

### ISSUE THREAD

**Issue by SirWilksBoothDaThird** [olivierfriard/BORIS#790]
Version: 8.27.8

Project integrity check: No issues found

Bug:
I've mentioned this in a previous bug report as well in a recent feature request, but I notice that, as of the latest version, pasting events doesn't seem to work.
I can copy events fine because the tabular data can be pasted as text in text editors, Microsoft Excel spreadsheets, etc. but pasting the copied event data within the BORIS 'Events' panel doesn't seem to do anything. I remember this feature actually working in one of the many BORIS 7.x or 8.x versions that I'd tested previously, but it is impossible to remember which one exactly.

I think that the PDF manual mentions this paste function as fully functional, but the BORIS version in the manual is 8.25.4 (tried it but get an error when opening a specific observation in an otherwise-working project). In any case, https://www.boris.unito.it/user_guide/coding/#copy-events also refers to the paste function as working as it should.

**To Reproduce**

Steps to reproduce the issue:
1. Add a STATE or POINT event at 00:00:01.000 seconds with multiple modifiers and a specific subject
2. Right-click on the created event inside the 'Events for "[Subject name here]"' panel and, in the context menu, click 'Copy events' command to copy the event cell/row
3. Move forward to a different time position, such as to 00:00:02.000 seconds
4. Right-click on the empty space inside the 'Events for "[Subject name here]"' panel and choose the command 'Paste events' in order to paste the copied event

**Expected behavior**

Once the space within the 'Events' panel has been right-clicked to call up the context menu and 'Paste events' command has been clicked, it is expected that a new row containing the copied event data will be added at the end of the list of events or at the current time position.

Pasting an event has worked for me once before when testing older BORIS versions, but I unfortunately cannot remember which version it was, as this feature seems to work in rare cases and I've tested a LOT of BORIS versions, both full and portable installs.
When it worked, it especially brought me much joy because, if I recall correctly, pasting a STATE event actually worked like an automated 'Fix unpaired events' function by "closing" the STATE event with the exact same properties that it was "opened" or started. This speeds up the workflow immensely by discarding the need for manually replicating the STATE event all over again when stopping/closing its state.

**Desktop:**
 - OS: Microsoft Windows 10 Pro (x64-based PC)
 - Both BORIS Portable and Full installations

**Comment by SirWilksBoothDaThird** [comment]
I'm interested to know if anyone else has this problem. I remember someone on the BORIS github mentioning that they are able to copy/paste events into the events table. Is this an old feature that just hasn't caught up to the latest BORIS version or am I just using it incorrectly?

**Comment by SirWilksBoothDaThird** [comment]
After checking over the changelog for all BORIS release to make sure that I didn't miss any changes to this function, I decided to try version 7.1.0 out of curiosity since it seems to be the first mention of copy/paste function implementation for events.
Granted, I tried the portable version, but, for the life of me, I have no idea why it is possible to copy an event to the clipboard but impossible to paste it into the events table. Either way, this maybe rules out the idea that this function was working before and became buggy after the 8.x release.

Either it is my system that is buggy, which would be strange given the ability to copy events and the otherwise flawless function of the latest BORIS versions, or I am missing something obvious. Perhaps all the columns -- including the 'Comment' column -- need to be filled in, I'm not sure, but it is a complete mystery and the log doesn't output any errors either when attempting to paste.
